Tasks/powerShell/powershell.ps1

```diff
@@ -0,0 +1,33 @@
+param (
+[string] $type,
+[string] $scriptName,
+[string] $arguments,
+[string] $workingFolder,
+[string] $script
+)
+
+Write-Verbose 'Entering powershell.ps1'
+Write-Verbose 'Current Working Directory is $cwd'
+Write-Host "Here I am "
+Write-Host "Type is" $type
+Write-Host "ScriptName is" $scriptName
+Write-Host "args are" $arguments
+Write-Host "Working folder is" $workingFolder
+Write-Host "Script is " $script
+
+if($workingFolder)
+{
+    if(!(Test-Path $workingFolder -PathType Container))
+    {
+        throw ("$workingFolder does not exist");
```

Comment:
This is user facing message and has to be localized. Use Get-LocalizedString and make relevant changes in resource file.